bolt-sidecar/src/state/execution.rs

```diff
@@ -123,6 +124,12 @@ impl ValidationError {
     }
 }
 
+impl From<pricing::PricingError> for ValidationError {
+    fn from(err: pricing::PricingError) -> Self {
+        Self::Internal(format!("Pricing error: {}", err))
+    }
+}
```

Comment:
this could also be done above as 

```rust
#[derive(Debug, Error)]
pub enum ValidationError {
	#[error("Pricing error: {0}")
	Pricing(#[from] pricing::PricingError)
```

returning "internal error" also means that bolt-rpc won't get feedback on why the commitment failed, which in my opinion we should provide for pricing errors. If this was intentional then it's also fine but let's discuss

Reply:
done